src/ru/skypro/Main.java

```diff
@@ -3,6 +3,79 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
-    }
-}
+        // write your code here
+        int clientOC = 0;
+        if (clientOC == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        } else if (clientOC == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        }
+
+
+        int clientDeviseYear = 2015;
+        if (clientOC == 1) if (clientDeviseYear >= 2015) {
```

Comment:
стоит объединить под одним условием, либо писать следующее уже с новой строки. Аналогично для условий ниже
Можете использовать ctrl+alt+L для автоформатирования
```suggestion
        if (clientOC == 1 && clientDeviseYear >= 2015) {
```